nrf_modem/include/nrf_modem.h

```diff
@@ -82,12 +83,67 @@ struct nrf_modem_shmem_cfg {
 	} trace;
 };
 
+/**
+ * @defgroup nrf_modem_fault_handling Modem fault handling
+ *
+ * @ingroup nrf_modem
+ * @{
+ */
+
+/** Undefined fault (e.g. undefined interrupt) */
+#define NRF_MODEM_FAULT_UNDEFINED 0x001
+/** HW WD reset detected */
+#define NRF_MODEM_FAULT_HW_WD_RESET 0x002
+/** Hard fault */
+#define NRF_MODEM_FAULT_HARDFAULT 0x003
+/** Memory management fault */
+#define NRF_MODEM_FAULT_MEM_MANAGE 0x004
+/** Bus fault */
+#define NRF_MODEM_FAULT_BUS 0x005
+/** Usage fault */
+#define NRF_MODEM_FAULT_USAGE 0x006
+/** Modem has been reset due to secure ctrl */
+#define NRF_MODEM_FAULT_SECURE_RESET 0x007
+/** Error handler has crashed */
+#define NRF_MODEM_FAULT_PANIC_DOUBLE 0x008
+/** Modem startup has failed four times in row */
+#define NRF_MODEM_FAULT_PANIC_RESET_LOOP 0x009
+/** Assert */
+#define NRF_MODEM_FAULT_ASSERT 0x010
+/** Unconditional SW reset */
+#define NRF_MODEM_FAULT_PANIC 0x011
+/** Flash erase fault */
+#define NRF_MODEM_FAULT_FLASH_ERASE 0x012
+/** Flash write fault */
+#define NRF_MODEM_FAULT_FLASH_WRITE 0x013
+/** Undervoltage fault */
+#define NRF_MODEM_FAULT_POFWARN 0x014
+/** Overtemperature fault */
+#define NRF_MODEM_FAULT_THWARN 0x015
+
+
+/** @brief Modem fault info struct. */
+
+struct nrf_modem_fault_info {
+	uint32_t reason;
```

Comment:
```suggestion
/** @brief Modem fault info struct. */
struct nrf_modem_fault_info {
	uint32_t reason;
```

Reply:
What's the suggestion? It's not clear from GitHub.

Reply:
delete blank line